src/components/Popover/Popover.scss

```diff
@@ -8,6 +8,10 @@ $block: '.#{$ns}popover';
     display: inline-block;
     position: relative;
 
+    &_disabled {
+        cursor: default;
+    }
+
```

Comment:
I am not sure if it is the correct order to put to so notify me if this need to be fixed

Reply:
What about inverting the rule: `:not(&_disabled) {cursor: pointer}`? So we don't override the cursor of underlying element in Popover whet it's disabled.

Reply:
Oh, ok

I slightly modified it by adding an `&` at the beggining like follows:

`&:not(&_disabled) {cursor: pointer}`

without it it was a mess